src/index.ts

```diff
@@ -28,29 +31,29 @@ module.exports = (options = {}) => {
     id: options.id,
     cert: options.cert
   })
-  const server = createServer({webhook, logger})
+  const server: express.Application = createServer({webhook, logger})
```

Comment:
`: express.Application` should be inferred here, which means you can drop the `import * as express from 'express'` above

Reply:
Yeh this one had me scratching my head. When i take it away, I get this:

`Exported variable 'createProbot' has or is using name 'e.Application' from external module "/Users/tcbyrd/ts-practice/probot/node_modules/@types/express/index" but cannot be named.`

So {createServer} is importing `express.Application`, but it gets aliased. I'm obviously missing something about how these types are being imported and exported.

Reply:
@tcbyrd yeah, let me wrap my head around what The Right Thing™ here is to do